notebooks/book2/02/sub_super_gauss_plot.ipynb

```diff
@@ -0,0 +1,463 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.        plt.yticks(jnp.arange(0, 0.55, step=0.05))
</pre><p>Due to this yticks looks very messy</p><p>At this point, it seems you are trying to make non-latexified figure more prettier, but Note that our ultimate goal is to make latexified figure neat and beautiful :) </p><p>It is okay if <code>linewidths</code> in <strong>non-latexified figure</strong> is too small or no. of <code>yticks</code> in <strong> non-latexified figure</strong> are less. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/63/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
okay, I understand the problem